scw/client_option.go

```diff
@@ -121,3 +121,12 @@ func WithDefaultZone(zone utils.Zone) ClientOption {
 		s.defaultZone = zone
 	}
 }
+
+// WithDefaultPageSize client option overrides the default page size of the SDK.
+//
+// It will be used as the default value of the page_size field in all requests made with this client.
+func WithDefaultPageSize(pageSize int32) ClientOption {
+	return func(s *settings) {
+		s.defaultPageSize = pageSize
```

Comment:
should check that page size is not 0

Reply:
The check should not happen here but in [`settings.validate()`](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/settings.go#L33)